rispy/parser.py

```diff
@@ -225,24 +275,19 @@ def loads(
 
     Args:
         obj (str): A string version of an RIS file.
-        mapping (Dict, optional): a tag mapping dictionary.
-        implementation (RisImplementation): RIS implementation; base by default.
-        strict (bool): Boolean to allow non-tag data between records to be ignored.
+        implementation (RisImplementation): RIS implementation; base by
+                                            default.
 
     Returns:
         list: Returns list of RIS entries.
     """
-
-    # remove BOM if present
-    obj = obj.lstrip("\ufeff")
-
-    filelines = obj.split("\n")
-
-    implementation = RisImplementation(implementation)
-
     if implementation == RisImplementation.WOK:
```

Comment:
suggest breaking API here (and documenting in PR) and passing a `Optional[BaseParser]` with a default of None if None, then the load could instantiate a `RisParser()` class.